orbit/pkg/table/software_update/software_update_test.go

```diff
@@ -36,5 +37,5 @@ func TestIsNewSoftwareAvailable(t *testing.T) {
 
 func TestColumns(t *testing.T) {
 	col := Columns()
-	require.Equal(t, []table.ColumnDefinition([]table.ColumnDefinition{table.ColumnDefinition{Name: "software_update_required", Type: "INTEGER"}}), col)
+	require.Equal(t, []table.ColumnDefinition([]table.ColumnDefinition{{Name: "software_update_required", Type: "INTEGER"}}), col)
```

Comment:
is this change intentional?

Reply:
it was as i was getting a listing error. Idk why i was getting it so tried to fix with this line. https://github.com/fleetdm/fleet/actions/runs/4564331900/jobs/8053962450?pr=10874. I'm thinking of reverting as it seems to break other things